impl/src/main/java/com/sun/faces/config/manager/DbfFactory.java

```diff
@@ -256,49 +431,57 @@ public FacesEntityResolver() {
                 }
             }
 
+
         } // END JsfEntityResolver
 
+
         // ----------------------------------------- Methods from DefaultHandler
 
+
         /**
-         * <p>
-         * Resolves the physical resource using the last segment of the <code>systemId</code> (e.g.
-         * http://java.sun.com/dtds/web-facesconfig_1_1.dtd, the last segment would be web-facesconfig_1_1.dtd). If a mapping
-         * cannot be found for the segment, then defer to the <code>DefaultHandler</code> for resolution.
-         * </p>
+         * <p>Resolves the physical resource using the last segment of
+         * the <code>systemId</code>
+         * (e.g. http://java.sun.com/dtds/web-facesconfig_1_1.dtd,
+         * the last segment would be web-facesconfig_1_1.dtd).  If a mapping
+         * cannot be found for the segment, then defer to the
+         * <code>DefaultHandler</code> for resolution.</p>
          */
         @Override
-        public InputSource resolveEntity(String publicId, String systemId) throws SAXException {
+        public InputSource resolveEntity(String publicId, String systemId)
+        throws SAXException {
 
-            // publicId is ignored. Resolution performed using
+            // publicId is ignored.  Resolution performed using
             // the systemId.
 
             // If no system ID, defer to superclass
             if (systemId == null) {
                 InputSource result;
                 try {
                     result = super.resolveEntity(publicId, null);
-                } catch (IOException | SAXException e) {
+                }
+                catch (IOException | SAXException e) {
```

Comment:
As much as I initially disliked the catch being on the same line of the closing brace, this is actually according to the Sun standard and the Jakarta EE standard (which inherits from Sun)